docs/userguide/006_The_Core_API.adoc

```diff
@@ -284,7 +284,7 @@ object from the Reflection API. For example:
 
 [source,java,options="nowrap"]
 ----
-JavaClasses classes = new ClassFileImporter().importClasspath(new ImportOptions());
+JavaClasses classes = new ClassFileImporter().importClasspath();
```

Comment:
L29 and L50, as well as `009_JUnit_Support.adoc`'s L107, still refer to
> `ImportOptions`

which could IMO be changed to
> `ImportOption`s

Reply:
Hmm, yes, I was pondering about it. Thought it also looks weird to have the 's' outside (and I think in other Javadocs, etc., I sometimes put the whole plural into the code as well), but maybe it's better to change it :thinking:

Reply:
A JavaDoc link is different because it unambigiously targets the `ImportOption` class.

But in the documentation, you don't have a link. When I see "`ImportOptions`" there, I'd assume that there is a class (or some other code construct) `ImportOptions` and be confused (if not even misled when I find the package-private class `ImportOptions`). I'd use "`ImportOption`s" to separate which part is code (`ImportOption`) and which part is grammar ("-s").